File: PRAK-5/PRAK502-2410817110011-MUHAMMADAZRIELAKBAR.py
def hitung(nilai1, nilai2) :
  hasil = nilai1 - nilai2
  return -hasil if hasil < 0 else hasil
def mutlak(angka) :
  return angka if angka>= 0 else -angka
def main() :
  a,b,c,d = map(int, input("").split())
  Hasil = hitung(a,c) + hitung(b, d)
  print(Hasil)
main()

File: PRAK-5/PRAK502-2410817110011-MUHAMMADAZRIELAKBAR.c
#include <stdio.h>
#include <math.h>
int hitung(int nilai1, int nilai2)
{
  int Hasil = nilai1 - nilai2;
  return Hasil < 0 ? -Hasil : Hasil;
}
int mutlak(int angka)
{
  return angka >= 0 ? angka : -angka;
}
int main()
{
  int a, b, c, d;
  scanf("%d", &a);
  scanf("%d", &c);
  scanf("%d", &b);
  scanf("%d", &d);
  int Hasil = hitung(a, b) + hitung(c, d);
  printf("%d", mutlak(Hasil));
}